frontend/src/routes/+layout.svelte

```diff
@@ -8,9 +8,14 @@
 	import { goto } from "$app/navigation";
 	import { userInfo } from "$lib/stores/user-info";
 	import YakManCard from "$lib/components/YakManCard.svelte";
+	import { page } from "$app/stores"
 
 	export let data;
 
+	const pagesWithNoRefreshTokenNeeded: string[] = [
+		'/session/oauth-callback'
+	]
+
 	$: {
```

Comment:
this is a hacky work around to make sure that `<slot />` gets rendered so that the oauth-callback works properly